controllers/keystoneendpoint_controller.go

```diff
@@ -287,15 +288,16 @@ func (r *KeystoneEndpointReconciler) reconcileNormal(
 	helper *helper.Helper,
 	os *openstack.OpenStack,
 ) (ctrl.Result, error) {
-	util.LogForObject(helper, "Reconciling Endpoint normal", instance)
+	logFromCtx := log.FromContext(ctx)
+	logFromCtx.Info("Reconciling Endpoint normal", "instance", instance)
 
 	//
 	// Wait for KeystoneService is Ready and get the ServiceID from the object
 	//
 	ksSvc, err := keystonev1.GetKeystoneServiceWithName(ctx, helper, instance.Spec.ServiceName, instance.Namespace)
 	if err != nil {
 		if k8s_errors.IsNotFound(err) {
-			util.LogForObject(helper, fmt.Sprintf("KeystoneService %s not found", instance.Spec.ServiceName), instance)
+			logFromCtx.Info(fmt.Sprintf("KeystoneService %s not found", instance.Spec.ServiceName), "instance", instance)
```

Comment:
* ditto, we don't need to log the instance manually
* also I would use structured logging here for the KeystoneService:
```
l.Info("KeystoneService not found", "KeystoneService", instance.Spec.ServiceName)
```